### PR TITLE
Test data added

### PR DESCRIPTION
**Pull request by guanqiaofeng**
Two sets of test data added. TEST_PRO passes all validation of schema. TEST_ERR has errors based on our schema. Details are in the readme file of each folder.

### CODE REVIEW

**Review by lindaxiang — COMMENTED** [review]
It's nice to have the `validate.py` code along with the test datasets. Thinking that if it's better to move `validate.py` and `dictionary.json` to seperate `test_data/script` folder instead of putting them into each dataset folder?

**Comment by guanqiaofeng** [comment]
@lindaxiang Thank you for the feedback! It is very good suggestion. I've implemented the changes.

File: test_data/TEST_PRO/readme.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+$ python3 ../script/validate.py
```
Comment (edsu7) [inline test_data/TEST_PRO/readme.txt:1]: Could add arguments field for directories containing TSVs or for individual TSVs

File: test_data/script/validate.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    print(f"All required columns present in {schema_name} schema: {cols_ok}")
+
+# Mapping between JSON schema types and pandas types
+type_mapping = {
+    'string': 'object',  # In pandas, strings are typically stored as objects
+    # Add more mappings as needed, e.g., 'integer': 'int64', 'number': 'float64', etc.
+}
+
+# Function to check if data matches the defined list (enumeration)
+def validate_code_list(column, code_list):
```
Comment (edsu7) [inline test_data/script/validate.py:26]: Return row with offending value?

**Review by lindaxiang — APPROVED** [review]
LGTM

File: test_data/TEST_ERR/readme.txt
Comment (guanqiaofeng) [inline test_data/TEST_ERR/readme.txt:1]: Address the issue https://github.com/Pan-Canadian-Genome-Library/data-dictionary/pull/3#discussion_r1556363473

File: test_data/script/validate.py [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+    invalid_rows = []
```
Comment (guanqiaofeng) [inline test_data/script/validate.py:29]: Row numbers are included in the error message. Addressed https://github.com/Pan-Canadian-Genome-Library/data-dictionary/pull/3#discussion_r1556364008

File: test_data/script/readme.txt
Comment (guanqiaofeng) [inline test_data/script/readme.txt:1]: Added arguments field for directories containing TSVs or for individual TSVs. Addressed https://github.com/Pan-Canadian-Genome-Library/data-dictionary/pull/3#discussion_r1556363473

**Review by edsu7 — APPROVED** [review]
lgtm

**Pull request merged**